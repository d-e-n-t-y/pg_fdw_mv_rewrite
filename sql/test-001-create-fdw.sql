-- create FDW server

DO $d$
    BEGIN
        EXECUTE $$CREATE SERVER loopback FOREIGN DATA WRAPPER pg_fdw_mv_rewrite
            OPTIONS (dbname '$$||current_database()||$$',
                port '$$||current_setting('port')||$$'
            )$$;
    END;
$d$;

alter server loopback options (add use_remote_estimate 'true');

CREATE USER MAPPING FOR public SERVER loopback;

create table pgx_rewritable_matviews (matviewschemaname name, matviewname name, tables text[]);
