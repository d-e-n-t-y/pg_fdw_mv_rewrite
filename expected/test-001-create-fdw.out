-- create rewrite mapping table
create table pgx_rewritable_matviews (matviewschemaname name, matviewname name, tables text[]);
